driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -265,12 +265,13 @@ fn write_input(input_file: &str, input: &str) -> std::io::Result<()> {
 
 fn run_solver(
     input_file: &str,
+    result_folder: &str,
     optimization_model: OptimizationModel,
 ) -> Result<(), PriceFindingError> {
     let optimization_model_str = optimization_model.to_args();
     let output = Command::new("python")
         .args(&["-m", "batchauctions.scripts.e2e._run"])
-        .arg(RESULT_FOLDER)
+        .arg(result_folder)
```

Comment:
Do we need to create the result folder being calling the solver?

Reply:
Good question! It's created by the solver.